### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ksiler**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [x] windows
- [ ] android

### Description of changes

- Adds additional bullet points to help verify if right dependencies are installed
- Remove dated screenshot under RNW Development Dependencies (screenshot featured older versions of required tools)
- Remove section referencing adding the App Signing Certificate to your Personal Certificate Store since this is no longer required (or even possible to do) after change #755 was merged

### Verification

Ran through the steps on clean machine (cloning repo and downloading dependencies from scratch)

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [x] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

**Review by Saadnajmi — APPROVED** [review]

File: apps/windows/README.md [hunk 1]
```diff
@@ -5,23 +5,14 @@
...
-1. Windows+R to open the Run Dialog and type in `certmgr.msc`. **NOTE:** For some reason this takes some time on my machine.
-2. Expand `Personal` and right click on `Certificates`
-3. Under `All tasks` click `Import`
-4. For the Current User, `Browse` to find `FluentTester_TemporaryKey.pfx` in your local clone of the repo.
-5. On the next page, type in 'password' for the password.
-   ![Certificate Manager Browse Window](./../../assets/CertFile.png)
-
 ## Running the App
 
 1. Run `yarn` from the root of the repo
```
Comment (Saadnajmi) [inline apps/windows/README.md:18]: I think there's actually a `yarn install-windows-test-app` or something in between these two steps

Comment (Saadnajmi) [reply apps/windows/README.md:18]: Ah, it's run as a pre step for the script "windows"
> "prewindows": "install-windows-test-app --use-nuget",

**Review by rurikoaraki — APPROVED** [review]

File: apps/windows/README.md (hunk 1 above)
Comment (ksiler) [reply apps/windows/README.md:18]: Thanks, I'll double check but I believe I was able to pull up the tester without it. If it is required, it sounds like it's something you only have to run once though? and not every time?

Comment (Saadnajmi) [reply apps/windows/README.md:18]: It has to run every time. But running `yarn windows` invokes `yarn prewindows`, ensuring it is run every time so we're good!

Comment (ksiler) [reply apps/windows/README.md:18]: Gotcha, didn't know that was a thing! Very cool.

**Pull request merged**